gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTable.java

```diff
@@ -113,7 +112,7 @@ public long getId() {
   }
 
   public static KnoxShellTableBuilder builder() {
-    return new KnoxShellTableBuilder(getUniqueTableId());
+    return new KnoxShellTableBuilder();
   }
 
   static long getUniqueTableId() {
```

Comment:
Probably doesn't need to be accessible anymore? Also would make more sense as a UUID instead of some long based on time? Doesn't really matter for this PR though

Reply:
What do you mean by `doesn't need to be accessible`? The `builder` method? That must be `public`; that is the entry point...one usually builds a table like this:
`locations = KnoxShellTable.builder().jdbc().driver("org.apache.derby.jdbc.EmbeddedDriver").connectTo("jdbc:derby:/Users/smolnar/test/derbyDb").sql("SELECT * FROM sample.locations where zip < 10")`

Reply:
`static long getUniqueTableId()` but looks like it is used in tests.

Reply:
Yes, I just wanted to reuse code...